pkg/queue/health/health_state.go

```diff
@@ -24,8 +24,11 @@ import (
 
 const (
 	// Return `queue` as body for 200 responses to indicate the response is from queue-proxy.
-	aliveBody    = "queue"
+	aliveBody = "queue"
+	// Return `queue not ready` as body for 503 response for queue not yet ready.
 	notAliveBody = "queue not ready"
+	// Return `shutting down` as body for 503 response if the queue got a shutdown request.
```

Comment:
nit: Comment here mentions its meant for 503 response.

Reply:
Ooops! Fixed now!